executor/join.go

```diff
@@ -419,11 +420,22 @@ func (e *HashJoinExec) join2Chunk(workerID uint, outerChk *chunk.Chunk, hCtx *ha
 		joinResult.err = err
 		return false, joinResult
 	}
+
+	hCtx.initHash(outerChk.NumRows())
+	for _, i := range hCtx.keyColIdx {
+		err = codec.HashChunkSelected(e.rowContainer.sc, hCtx.hashVals, outerChk, hCtx.allTypes[i], i, hCtx.buf, hCtx.hasNull, selected)
```

Comment:
```suggestion
		err = codec.HashChunkSelected(e.ctx, hCtx.hashVals, outerChk, hCtx.allTypes[i], i, hCtx.buf, hCtx.hasNull, selected)
```

Reply:
The implementation of `HashChunkColumns` relies on `HashChunkSelected`.
It is inconvenient to pass `sessionctx.Context` from `HashChunkColumns`.